db/commitlog/commitlog.cc

```diff
@@ -674,7 +693,7 @@ class db::commitlog::segment : public enable_shared_from_this<segment>, public c
         auto now = clock_type::now();
         auto ms = std::chrono::duration_cast<std::chrono::milliseconds>(
                 now - _sync_time).count();
-        if ((_segment_manager->cfg.commitlog_sync_period_in_ms * 2) < uint64_t(ms)) {
+        if (_segment_manager->cfg.commitlog_sync_period_in_ms <= uint64_t(ms)) {
```

Comment:
Is this a separate change?

Reply:
No, not really. We are however repurposing this method to actually check if we are exactly or over sync time. Previously it was only used from the alloc-path (not the periodic task), to check if we should intervene-sync because the periodic lagged. Now it instead used solely from the actual message loop sync code.